docs/source/query.rst

```diff
@@ -236,37 +282,38 @@ For example, instead of using the regex-operator, as shown above, we could write
 
 .. code-block:: python
 
-    {"protocol": {"$where": 'lambda x: "assembly" in x'}}
+    project.find_jobs({"protocol": {"$where": 'lambda x: "assembly" in x'}})
 
+.. code-block:: bash
+
+    signac find '{"protocol": {"$where": "lambda x: \"assembly\" in x"}}'
 
 .. _simplified-filter:
 
 Simplified Syntax on the Command Line
 =====================================
 
-It is possible to use search expressions directly on the command line, for example in combination with the ``$ signac find`` command.
-In this case filter arguments are expected to be provided as valid JSON expressions.
 For simple filters, you can use a simplified syntax instead of writing JSON.
-For example, instead of ``{'p': 2}``, you can type ``p 2``.
+For example, instead of ``signac find '{"p": 2}'``, you can type ``signac find p 2``.
 
 A simplified expression consists of key-value pairs in alternation.
 The first argument will then be interpreted as the first key, the second argument as the first value, the third argument as the second key, and so on.
 If you provide an odd number of arguments, the last value will default to ``{'$exists': true}``.
+
 Querying via operator is supported using the ``.``-operator.
 Finally, you can use ``/<regex>/`` instead of ``{'$regex': '<regex>'}`` for regular expressions.
 
-The following list shows simplified expressions on the left and their equivalent standard expression on the right.
+The following list shows simplified ``signac find`` expressions on the left, full query syntax in Python in the middle, and full query syntax in JSON on the right.
```

Comment:
Should go simplified JSON and Python since the simplified and JSON are both CLI and Python well is part of the Python API.